index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en-us">
+
+<!--Head section of HTML-->
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title> Horiseon </title>
+</head>
+
+<!--Body section of HTML-->
+
+<body>
+    <!--Header section-->
+    <header class="header">
+        <h1>Hori<span class="seo">seo</span>n</h1>
+        <!--Listless navigation bar-->
+        <nav>
+            <ul>
+                <a class="nav-btn" href="#search-engine-optimization">Search Engine Optimization</a>
+                <a class="nav-btn" href="#online-reputation-management">Online Reputation Management</a>
+                <a class="nav-btn" href="#social-media-marketing">Social Media Marketing</a>
+            </ul>
+        </nav>
+    </header>
+
+    <!--Hero section, to declare an oversized banner at the top of the web page-->
+    <section class="hero"></section>
+
+    <!--Main section of web page-->
+    <main>
+        <!--Article section, will consist of 3 divs, seperating each content-section to contain img, h2 and p elements-->
+        <article class="content">
+            <!--Search Engine Optimization section-->
+            <div id="search-engine-optimization" class="content-section">
+                <img src="./assets/images/search-engine-optimization.jpg" class="float-left content-img" alt="Office utlities on a desk" />
+                <h2>Search Engine Optimization</h2>
+                <p>
+                    The dominance of mobile internet use means that users are searching for the right business as they travel, shop, or sit on their couch at home. Search Engine Optimization (SEO) allows you to increase your visibility and find the right customers for your
+                    business.
+                </p>
+            </div>
+            <!--Online Reputation Management section-->
+            <div id="online-reputation-management" class="content-section">
```

Comment:
This `div` tag could of been a `section` tag. This would of made it more semantic.